api/alembic/versions/cacd97331dcb_french_bar_to_c7.py

```diff
@@ -0,0 +1,80 @@
+"""french bar to c7
+
+Revision ID: cacd97331dcb
+Revises: ecdace5bfc4f
+Create Date: 2025-04-07 10:46:35.571916
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.orm.session import Session
+
+# revision identifiers, used by Alembic.
+revision = 'cacd97331dcb'
+down_revision = 'ecdace5bfc4f'
+branch_labels = None
+depends_on = None
+
+planning_weather_stations_table = sa.Table('planning_weather_stations', sa.MetaData(),
+                                           sa.Column('id', sa.Integer),
+                                           sa.Column('fuel_type_id', sa.Integer),
+                                           sa.Column('planning_area_id', sa.Integer),
+                                           sa.Column('station_code', sa.Integer),
+                                           sa.Column('order_of_appearance_in_planning_area_list', sa.Integer),
+                                           sa.Column('create_user', sa.String),
+                                           sa.Column('create_timestamp', sa.TIMESTAMP),
+                                           sa.Column('update_user', sa.String),
+                                           sa.Column('update_timestamp', sa.TIMESTAMP),
+                                           sa.Column('is_deleted', sa.Boolean))
+
+fuel_types_table = sa.Table('fuel_types', sa.MetaData(),
+                            sa.Column('id', sa.Integer),
```

Comment:
Just an FYI, you can omit all the column names you don't reference in your queries.